slave/cracker.go

```diff
@@ -0,0 +1,31 @@
+package slave
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+	"github.com/aaronang/cong-the-ripper/slave/brutedict"
+	"github.com/aaronang/cong-the-ripper/slave/hasher"
+)
+
+func Execute(task lib.Task, slave *Slave) {
+	bd := brutedict.New(&task)
+	digest, _ := b64.StdEncoding.DecodeString(task.Digest)
```

Comment:
No, I do not think your point stands if I am correct. So the `key` function gives me a `[]byte`. I do not want to transform this into something else, but want to compare it directly. But I cannot compare it with the `digest []byte` that you suggested. I still need to convert it back to a string and base64 decode it to compare it to the `key`